validation/telemetry/deletion-request.4.payload.pass.json

```diff
@@ -15,13 +15,9 @@
     "channel": "default"
   },
   "clientId": "c4582ba1-79fc-1f47-ae2a-671118dccd8b",
-  "payload": { 
-    "processes": {
-      "parent": {
-        "scalars": {
-          "browser.engagement.unfiltered_uri_count": 782
-        }
-      }
+  "payload": {
+    "scalars": {
+      "browser.engagement.unfiltered_uri_count": 782
```

Comment:
Based on the schema I would expect this scalar to be located at `payload.scalars.parent`, and this is only valid because `"additionalProperties": false` isn't set on `scalars`.

Reply:
Good point! This is a bug in this test file.